src/ZODB/interfaces.py

```diff
@@ -818,6 +650,137 @@ def tpc_vote(transaction):
 
         """
 
+    def close():
```

Comment:
No, mostly a turd from an aborted attempt to factor the interfaces. I'll put it back.